src/libraries/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.WriteProperties.Bytes.cs

```diff
@@ -140,7 +140,7 @@ private void WriteBase64EscapeProperty(ReadOnlySpan<char> propertyName, ReadOnly
             int length = JsonWriterHelper.GetMaxEscapedLength(propertyName.Length, firstEscapeIndexProp);
 
             Span<char> escapedPropertyName = length <= JsonConstants.StackallocThreshold ?
-                stackalloc char[length] :
+                stackalloc char[JsonConstants.StackallocThreshold] :
```

Comment:
Are we happy with using the same threshold for chars, effectivly doubling the stackalloc size? I know the thresholds are all over the place in this repo (100,256,512,1024), but I'm still a bit uneasy with 512 bytes (not even talking about 1024 in MultiPartcontent).

Reply:
I'd prefer if the threshold were a fixed number of bytes, scaled accordingly for chars.  That said, as you say we're very inconsistent; we've talked about going through and standardizing everything across the repo, but haven't done so.

Reply:
Updated to have a scaled threshold for char allocations.